docs/querying/sql-data-types.md

```diff
@@ -34,12 +34,23 @@ Druid associates each column with a specific data type. This topic describes sup
 
 Druid natively supports the following basic column types:
 
-* LONG: 64-bit signed int
-* FLOAT: 32-bit float
-* DOUBLE: 64-bit float
-* STRING: UTF-8 encoded strings and string arrays
-* COMPLEX: non-standard data types, such as nested JSON, hyperUnique and approxHistogram, and DataSketches
-* ARRAY: arrays composed of any of these types
+* `LONG`: 64-bit signed int
+* `FLOAT`: 32-bit float
+* `DOUBLE`: 64-bit float
+* `STRING`: UTF-8 encoded strings and string arrays
+* `ARRAY`: arrays composed of any of these types
+
+## Complex types
+
+Druid natively supports the following complex types:
+* `COMPLEX<JSON>`: stores a copy of structured data in JSON format and specialized internal columns and indexes for nested basic types. Click here to learn more about [`COMPLEX<JSON>`](nested-columns.md)
+* `cardinality`: Data structure to compute the cardinality of Apache Druid dimensions using the HyperLogLog algorithm. Click here to learn more about [`cardinality`](hll-old.md#cardinality-aggregator)
```

Comment:
cardinality is an aggregator type, not a column type. it builds into a `COMPLEX<hyperUnique>` if stored in a column

Reply:
To clarify, is there both a COMPLEX<hyperUnique> and a hyperUnique aggregator? Because the [hll-old.md](https://druid.apache.org/docs/latest/querying/hll-old#hyperunique-aggregator ) page refers to a HyperUnique Aggregator?

Reply:
That page _only_ describes the aggregator types which are specified on native json queries (SQL handles all of this automatically during planning). I believe most extension pages will be like this, describing the aggregator types you would use in native json queries, and describing SQL function signatures, but not describing how the type appears when ingested into a table.

To clarify in this example, the `cardinality` and `hyperUnique` aggregators both spit out the type `COMPLEX< hyperUnique>`, the cardinality aggregator builds them from strings or numbers, and the hyperUnique aggregator merges them together